easybuild/easyconfigs/o/Optuna/Optuna-4.1.0-foss-2024a.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PythonBundle'
+
+name = 'Optuna'
+version = '4.1.0'
+
+homepage = "https://optuna.org/"
+description = """Optuna is an automatic hyperparameter optimization software framework,
+particularly designed for machine learning. It features an imperative,
+define-by-run style user API. Thanks to our define-by-run API, the code written
+with Optuna enjoys high modularity, and the user of Optuna can dynamically
+construct the search spaces for the hyperparameters."""
+
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('PyYAML', '6.0.2'),
+    ('SciPy-bundle', '2024.05'),
+    ('tqdm', '4.66.5'),
+    ('SQLAlchemy', '2.0.36'),
+    ('matplotlib', '3.9.2'),  # optional
+    ('plotly.py', '5.24.1'),  # optional
+    ('redis-py', '5.1.1'),  # optional
+    ('scikit-learn', '1.5.2'),  # optional
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    # optional
+    ('cmaes', '0.11.1', {
+        'checksums': ['cf71fa3679814723be771f2c9edd85f465b1bc1e409e1ad6d8a9e481efcd5160'],
+    }),
+    ('colorlog', '6.9.0', {
+        'checksums': ['bfba54a1b93b94f54e1f4fe48395725a3d92fd2a4af702f6bd70946bdc0c6ac2'],
+    }),
+    ('optuna', version, {
+        'use_pip_extras': 'optional',
```

Comment:
I think this should be removed. It doesn't seem to do anything for source builds and if it would it would install PyTorch: https://github.com/optuna/optuna/blob/1934ea79410ba5fb745f67699ed92ef6b0981653/pyproject.toml#L92

If I understand this correctly it should be used like `pip install optuna[optional]` but here it is `pip install .[optional]` which sees to ignore the option.   
It also is only required to request installing additional packages which we do manually here anyway.

Reply:
OK, done